.github/workflows/on-pr-merge.yml

```diff
@@ -0,0 +1,29 @@
+name: Bump version and publish release
+on:
+  workflow_dispatch: {}
```

Comment:
do you want people to be able to run this from the Actions page without pushing code to main?

Reply:
Yes, exactly. In case of a manual release, I think it would be slightly better to trigger a GH action than to run a script on your laptop.